### PR TITLE
feat: covers push functionality

### LINKED ISSUE #1331: cover push functionality in integration test
The relay offers functionality to inform wallets when they need to sign txs. The functionality currently isn't covered by our integ tests.

### PR DESCRIPTION
**Pull request by arein**
# Description

Covers push/prompt functionality

Resolves #1331

## How Has This Been Tested?

Ran the test

### CODE REVIEW

**Review by bkrem — APPROVED** [review]
💪

**Review by ganchoradkov — APPROVED** [review]

File: packages/sign-client/test/sdk/push.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    // Send a message which triggers the webhook to be invoked
+    const eventPayload: any = {
+      topic: sessionA.topic,
+      ...TEST_EMIT_PARAMS,
+    }
+    await clients.A.emit(eventPayload);
+    
+    // Validate webhook was called
+    const res = await axios.get(`${TEST_WEBHOOK_ENDPOINT}/${sessionA.topic}`);
```
Comment (HarryET) [inline packages/sign-client/test/sdk/push.spec.ts:38]: Might be worth having some retry here? In-case there is a delay on the Relay end?

**Review by HarryET — CHANGES_REQUESTED** [review]

**Review by xzilja — APPROVED** [review]

File: packages/sign-client/test/sdk/push.spec.ts (hunk 1 above)
Comment (arein) [reply packages/sign-client/test/sdk/push.spec.ts:38]: I first had this here but then thought YAGNI/KISS. I'll re-add if we see this fail for that reason

Comment (HarryET) [reply packages/sign-client/test/sdk/push.spec.ts:38]: :+1: Will Approve

**Review by HarryET — APPROVED** [review]

**Pull request merged**